indigo_app/urls.py

```diff
@@ -23,6 +23,7 @@
     url(r'^places/(?P<place>[a-z]{2}(-[^/]+)?)/activity$', places.PlaceActivityView.as_view(), name='place_activity'),
 
     url(r'^places/(?P<place>[a-z]{2}(-[^/]+)?)/tasks/$', tasks.TaskListView.as_view(), name='tasks'),
+    url(r'^places/(?P<place>[a-z]{2}(-[^/]+)?)/tasks(?P<frbr_uri>/\S+?)/$', tasks.TaskListView.as_view(), name='tasks_on_work'),
```

Comment:
This is going to get confusing -- which URL do we submit the filter form to? The one above or this one?

Let's use the normal task listing url, but include the `frbr_uri` as a query string field, like any other filter field.

Otherwise, we're essentially adding a new view entirely.

Reply:
we would be adding a new (template) view, but we'd be using the same (view) controller – and if our only aim was to view all tasks on one work I think it would work well?
I'm not sure I get the use case for filtering on multiple works, and thought this would be the simplest way of making sure that we 
a) display visually that we're looking at tasks on a particular work and 
b) retain the FRBR URI of that work in the context without having to pass it along unnecessarily for every other task listing view

Reply:
I think I misunderstood the concern here – are you talking about a filter form that doesn't exist yet?